module/Report/src/Model/Member.php

```diff
@@ -504,8 +504,11 @@ public function setPaid(int $paid): void
      *
      * @return string|null
      */
-    public function getIban(): ?string
+    public function getIban($print = false): ?string
     {
+        if ($print) {
+            return preg_replace('/(\\w{4})/', '$1 ', $this->iban);
+        }
```

Comment:
And here.

Reply:
[8fb8edb](https://github.com/GEWIS/gewisdb/pull/200/commits/8fb8edb178737bd2be7cd72116765a9b0f1725f2)